docs/node-mixin/dashboards/use.libsonnet

```diff
@@ -0,0 +1,200 @@
+        .addPanel(
+          g.panel('CPU Utilisation') +
+          g.queryPanel(|||
+            (
+              instance:node_cpu_utilisation:avg_rate1m
+            *
+              instance:node_num_cpu:sum
+            / ignoring (instance) group_left
+              sum without (instance) (instance:node_num_cpu:sum)
+            )
+          |||, '{{instance}}', legendLink) +
+          g.stack +
+          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) },
+        )
+        .addPanel(
+          // TODO: Is this a useful panel?
+          g.panel('CPU Saturation (load1 per CPU)') +
+          g.queryPanel(|||
+            (
+              instance:node_load1_per_cpu:ratio
+            / ignoring (instance) group_left
+              count without (instance) (instance:node_load1_per_cpu:ratio)
+            )
+          |||, '{{instance}}', legendLink) +
+          g.stack +
+          // TODO: Does `max: 1` make sense? The stack can go over 1 in high-load scenarios.
+          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) },
+        )
+      )
+      .addRow(
+        g.row('Memory')
+        .addPanel(
+          g.panel('Memory Utilisation') +
+          g.queryPanel('instance:node_memory_utilisation:ratio', '{{instance}}', legendLink) +
+          g.stack +
+          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) },
+        )
+        .addPanel(
+          g.panel('Memory Saturation (Swap I/O)') +
+          g.queryPanel('instance:node_memory_swap_io_bytes:sum_rate', '{{instance}}', legendLink) +
+          g.stack +
+          { yaxes: g.yaxes('Bps') },
+        )
+      )
+      .addRow(
+        g.row('Disk')
+        .addPanel(
+          g.panel('Disk IO Utilisation') +
+          // Full utilisation would be all disks on each node spending an average of
+          // 1 second per second doing I/O, normalize by metric cardinality for stacked charts.
+          g.queryPanel(|||
+            (
+              instance:node_disk_io_time:sum_rate1m
+            / ignoring (instance) group_left
+              count without (instance) (instance:node_disk_io_time:sum_rate1m)
+            )
+          |||, '{{instance}}', legendLink) +
+          g.stack +
+          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) },
+        )
+        .addPanel(
+          g.panel('Disk IO Saturation') +
+          g.queryPanel(|||
+            (
+              instance:node_disk_io_time_weighted:sum_rate1m
+            / ignoring (instance) group_left
+              count without (instance) (instance:node_disk_io_time_weighted:sum_rate1m)
+            )
+          |||, '{{instance}}', legendLink) +
+          g.stack +
+          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) },
+        )
+      )
+      .addRow(
+        g.row('Network')
+        .addPanel(
+          g.panel('Net Utilisation (Transmitted)') +
+          g.queryPanel('instance:node_net_utilisation:sum_irate', '{{instance}}', legendLink) +
```

Comment:
This is a rate1m now

Reply:
In fact, I haven't changed this whole row to the new network rules (with transmit and receive separate). Will rework.